### ISSUE TITLE
Práctico 5

### ISSUE THREAD

**Issue by jaarac** [Emanuel-Piovano/VisionPorComputadora#2]
Se debe guardar el rectángulo seleccionado en una nueva imagen, no la imagen  con el rectángulo.

**Comment by Emanuel-Piovano** [comment]
Perdón, no comprendí. Una vez seleccionado un rectángulo en la imágen, y cuando se apreta la letra g ¿ahí tiene que recortar la imágen según el rectángulo seleccionado y guardar esa imagen seleccionada? ¿En la imagen guardada no se ve el rectángulo marcado o sí?

**Comment by Emanuel-Piovano** [comment]
Ahí en base a su comentario y al comentario que hice anteriormente según lo que entendí en cuanto a la corrección que debo hacer, modifiqué el archivo del Práctico 5 y cree y subí los archivos Práctico 5-1 y Práctico 5-2, ya que no sabía bien cómo debía ser, por lo cual pensé subir estos dos programas para que los mire y me diga si es como alguno de estos dos que había que hacerlo. En caso de que alguno de estos esté bien, después daría de baja acá en el repositorio el Práctico 5 y alguno de estos dos y re nombraría al que esté bien como Práctico 5. 
Si no le pido que me vuelva a explicar como es para entender qué es lo que debo hacer.

**Comment by jaarac** [comment]
No, justamente la idea de un repositorio es que vayas editando siempre el mismo archivo y lo vayas subiendo. git se encarga de ir guardando las versiones para que no se pierda lo antiguo. Hoy me pasaron este tutorial de github, a lo mejor te sirve: https://drive.google.com/open?id=1S7fD1M0_jp0O9UXtTGqV8o3dYuwNsSaJ
La idea mía de que usen git, es por comodidad para subir ustedes los archivos y yo corregir. Es difícil que se caiga github o gitlab, en cambio el campus es super lento y a veces se cae. Además de esto, es una herramienta muy útil que se usa en la mayoría de las empresas.

**Comment by Emanuel-Piovano** [comment]
Ahí elimine los dos archivos y edité el práctico 5, agregué dos ediciones. La primera edición es cuando se oprime la letra "g" una vez seleccionado con un rectángulo sobre la imagen original, se recorta la imagen y se guarda como una imagen nueva. En esta aparece lo que se encontraba dentro del rectángulo más un marco que es el propio rectángulo.
La segunda edición, es que al oprimir la letra "g" una vez seleccionado con un rectángulo en la imagen original, se recorta la imagen en una imagen nueva, en este caso no figura el marco propio del rectángulo marcado en la imagen original. No sé de cuál de ambas formas debe ser el programa
